### DISCUSSION TITLE
UUID working for exclude baltop

### DISCUSSION THREAD

**Discussion by elRobz** [Zrips/CMI#5784] [Feature Requests]
Hello, my staff keeps changing their minecraft names, which is ok. but we are excluded for baltop, and have to change it manually all the time, is it any problems of you making it so we can use UUID? it might be other places UUID isnt working either.

**Comment by CloudeLecaw** [comment]
You can use the plceholder %cmi_user_uuid%  that was added with CMI 8.8.4.9

well i know its not listed in the wiki...
